### ISSUE TITLE
People say that our meta pages design is crap

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/meta#88]
After receiving a first feed in #50 and from @malev I have asked several programming communities about our design.

Everyone agrees that it looks like crap. Which is not good.

So, I guess we will put a redesign as a highest priority task.

Friendly ping to @AndreyErmilov

**Comment by NickVolynkin** [comment]
> Everyone agrees that it looks like crap

I disagree. The page looks like a nice and tidy readme on GitHub, which is good.
